src/styles/stopwatch.scss

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  height: $stopwatchRadius;
+  width: $stopwatchRadius;
+  border-radius: $circle;
+  border: 1vmin dotted black;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+
+    @include centererd-hands;
```

Comment:
It seems there is a typo in the mixin name `centererd-hands`. It should be corrected to `centered-hands` to improve readability and avoid confusion.